### PR TITLE
Publish process model as part of CloudProcessDeployedEvent

### PR DESCRIPTION
**Pull request by erdemedeiros**
Related to https://github.com/Activiti/Activiti/issues/2389

### CODE REVIEW

**Comment by erdemedeiros** [comment]
Depends on:
- https://github.com/Activiti/activiti-api/pull/89
- https://github.com/Activiti/Activiti/pull/2549
- https://github.com/Activiti/activiti-cloud-api/pull/71

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/pom.xml [hunk 1]
```diff
@@ -99,6 +99,10 @@
       <groupId>org.springframework.cloud</groupId>
       <artifactId>spring-cloud-stream</artifactId>
     </dependency>
+    <dependency>
```
Comment (ryandawsonuk) [inline activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/pom.xml]: I don't see this being used in any of the classes in the PR. Was it just missing already or is it used in the background somehow? I tried taking this out and doing a build and it doesn't seem to be needed for compilation or unit tests (haven't tried ITs yet though).

Comment (ryandawsonuk) [reply activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/pom.xml]: ITs passed too actually

Comment (erdemedeiros) [reply activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/pom.xml]: Good spot @ryandawsonuk, it was initially used by a piece of code that I refactored later on and it's no longer used. I'll remove it.

**Review by ryandawsonuk — APPROVED** [review]

**Review by ryandawsonuk — APPROVED** [review]

**Pull request merged**